src/components/Modals/index.js

```diff
@@ -10,4 +10,4 @@ import UploadDocumentModal from './UploadDocumentModal';
  * @namespace Modals
  */
 
-export { AddClientModal, DeleteClientModal, LogoutModal, NewMessageModal, UploadDocumentModal };
+export { AddContactModal, NewMessageModal, DeleteContactModal, LogoutModal, UploadDocumentModal };
```

Comment:
I'd prefer alphabetical order, but not an issue.  Just find it easier for future maintainability/changes.

Reply:
fixed